test/libsolidity/semanticTests/array/copying/array_copy_including_array.sol

```diff
@@ -33,6 +33,8 @@ contract c {
         return (small.length, large.length);
     }
 }
+// ====
+// compileViaYul: also
```

Comment:
This test works locally for you?

Reply:
I guess it worked when it was still rebased on top of another PR?

Reply:
Probably, looks like it is solved now